connectors/rdbms/r2dbc/src/test/kotlin/io/github/gabfssilva/river/r2dbc/R2dbcFlowExtTest.kt

```diff
@@ -0,0 +1,57 @@
+package io.github.gabfssilva.river.r2dbc
+
+import io.kotest.core.spec.style.FeatureSpec
+import io.r2dbc.spi.ConnectionFactories
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.reactive.awaitFirst
+
+class R2dbcFlowExtTest : FeatureSpec({
+    feature("Test statement execution") {
+        val connection = h2Client.awaitFirst()
+
+        scenario("Insert statements") {
+            val flow = (1..10)
+                .asFlow()
+                .map {
+                    connection.createStatement("INSERT INTO test (id) VALUES ('$it')")
+                }
+
+            connection.executeStatementFlow(flow).collect(::println)
```

Comment:
Does it return something? I think we could make some assertions to be sure it's working properly.

Reply:
Yes, it do. The `Result` object is returned in all type of statements